runtime/linker-scripts/jolt.x

```diff
@@ -2,8 +2,8 @@ ENTRY(_start);
 
 SECTIONS
 {
-  __memory_top = 0x80400000;
-  . = 0x80000000;
+  __memory_top = 0x400000;
```

Comment:
This change isn't consistent with the Jolt prover.

Reply:
reverted in 0944f29